prod_validators.py

```diff
@@ -0,0 +1,810 @@
+# coding: utf-8
```

Comment:
Why did you create this file and why did you not modify the core/domain/prod_validators.py?

Reply:
yes, it was my mistake 
Can I now modify the changes in core/domain/prod_validators.py and delete the created prod_validators.py and commit the changes from the repository set locally?